lib/taskana-core/src/test/java/acceptance/task/UpdateTaskAccTest.java

```diff
@@ -136,64 +141,57 @@ public void testThrowsExceptionIfTaskHasAlreadyBeenUpdated()
-        // Task updatedTask = taskService.updateTask(task);
-        //
-        // assertNotNull(updatedTask);
-        // assertEquals("T2100", updatedTask.getClassification().getKey());
-        // assertNotEquals(updatedTask.getCreated(), updatedTask.getModified());
-        // assertEquals(22, updatedTask.getPriority());
-        // assertEquals(task.getPlanned(), updatedTask.getPlanned());
-        // assertEquals(???, updatedTask.getDue()); // should be one day later
-        // assertEquals("T-Vertragstermin VERA", updatedTask.getName());
-        // assertEquals("T-Vertragstermin VERA", updatedTask.getDescription());
+        throws TaskNotFoundException, WorkbasketNotFoundException, ClassificationNotFoundException,
+        InvalidArgumentException, ConcurrencyException, InvalidWorkbasketException, NotAuthorizedException {
+
+        TaskService taskService = taskanaEngine.getTaskService();
+        Task task = taskService.getTask("TKI:000000000000000000000000000000000000");
+        ClassificationSummary classificationSummary = task.getClassificationSummary();
+        task.setClassificationKey("T2100");
+        Task updatedTask = taskService.updateTask(task);
+
+        assertNotNull(updatedTask);
+        assertEquals("T2100", updatedTask.getClassificationSummary().getKey());
+        assertThat(updatedTask.getClassificationSummary(), not(equalTo(classificationSummary)));
+        assertNotEquals(updatedTask.getCreated(), updatedTask.getModified());
+        assertEquals(task.getPlanned(), updatedTask.getPlanned());
+        assertEquals(task.getName(), updatedTask.getName());
+        assertEquals(task.getDescription(), updatedTask.getDescription());
     }
 
-    @Ignore
     @WithAccessId(
         userName = "user_1_1",
-        groupNames = {"group_1"})
+        groupNames = { "group_1" })
     @Test
     public void testCustomPropertiesOfTask()
-        throws SQLException, NotAuthorizedException, InvalidArgumentException, ClassificationNotFoundException,
-        WorkbasketNotFoundException, TaskAlreadyExistException, InvalidWorkbasketException {
-
-        // TaskService taskService = taskanaEngine.getTaskService();
-        // Task with classification T2000
-        // Task task = taskService.getTask("TKI:000000000000000000000000000000000000");
-        // task.setCustomProperty1("T2100"));
-        // ...
-        // Task updatedTask = taskService.updateTask(task);
-        //
-        // assertNotNull(updatedTask);
-        // meaningful assertions
+        throws TaskNotFoundException, WorkbasketNotFoundException, ClassificationNotFoundException,
+        InvalidArgumentException, ConcurrencyException, InvalidWorkbasketException, NotAuthorizedException {
+        TaskService taskService = taskanaEngine.getTaskService();
+        Task task = taskService.getTask("TKI:000000000000000000000000000000000000");
+        task.setCustom1("T2100");
+        Task updatedTask = taskService.updateTask(task);
```

Comment:
Please get the task again for assertion of the test

Reply:
DONE.
Updated this for all update() Results and added fail() on testThrowsExceptionIfMandatoryPrimaryObjectReferenceIsNotSetOrIncomplete(), because there wouldn´t be a failure when there is now exception thrown in the tested cases.